tests/Curl/CurlTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brick\Std\Tests\Curl;
+
+use Brick\Std\Curl\Curl;
+use PHPUnit\Framework\TestCase;
+
+class CurlTest extends TestCase
+{
+    public function testConstructorShouldInitialCurl()
+    {
+        $curl = new Curl();
+
+        $this->assertInternalType('object', $curl);
+    }
+
+    public function testCurlClassCanBeCloned()
+    {
+        $curl = new Curl();
+        $newCurl = clone $curl;
+
+        $this->assertInstanceOf(Curl::class, $newCurl);
+    }
+
+    public function testGetInfoWithSpecificOption()
+    {
+        $curl = new Curl();
+
+        $this->assertSame(0, $curl->getInfo(CURLINFO_HTTP_CODE));
+    }
+
+    public function testGetInfo()
+    {
+        $expectedArray = [
+            'url' => '',
+            'content_type' => null,
+            'http_code' => 0,
+            'header_size' => 0,
+            'request_size' => 0,
+            'filetime' => 0,
+            'ssl_verify_result' => 0,
+            'redirect_count' => 0,
+            'total_time' => 0.0,
+            'namelookup_time' => 0.0,
+            'connect_time' => 0.0,
+            'pretransfer_time' => 0.0,
+            'size_upload' => 0.0,
+            'size_download' => 0.0,
+            'speed_download' => 0.0,
+            'speed_upload' => 0.0,
+            'download_content_length' => -1.0,
+            'upload_content_length' => -1.0,
+            'starttransfer_time' => 0.0,
+            'redirect_time' => 0.0,
+            'redirect_url' => '',
+            'primary_ip' => '',
+            'certinfo' => [],
+            'primary_port' => 0,
+            'local_ip' => '',
+            'local_port' => 0,
+        ];
+        $curl = new Curl();
+
+        $this->assertSame($expectedArray, $curl->getInfo());
+    }
+
+    public function testSetOptionShouldBeSet()
+    {
+        $curl = new Curl();
+
+        $this->assertNull($curl->setOption(CURLOPT_URL, 'http://exampl.ecom'));
```

Comment:
No need to assert anything as it returns `void`. What would be good however, is to call `getOption()` and check that the value has been set. Oh, and there's a typo in 'exampl.ecom'.

Reply:
I will fix the typo "example.com" and use the ```getOption``` method to do the assertions.

Reply:
BTW, the ```getOption``` seems to be not implemented in this ```Curl``` class.
Should I implement this method by myself?

Reply:
Perhaps we just use the ```getInfo``` to solve this?

Reply:
The ```curl_getinfo``` will return the associate array, but the ```getInfo``` parameter is set the integer.
I will let parameter be the string.

Reply:
I misunderstand the ```curl_getinfo``` and it seems that the ```curl_setopt``` options is different from ```curl_getinfo``` options.